openslides_backend/action/action.py

```diff
@@ -37,7 +37,15 @@
 from ..shared.interfaces.logging import LoggingModule
 from ..shared.interfaces.services import Services
 from ..shared.interfaces.write_request import WriteRequest
-from ..shared.patterns import Collection, FullQualifiedId, transform_to_fqids
+from ..shared.patterns import (
+    FullQualifiedId,
+    fqfield_collection,
+    fqfield_field,
+    fqfield_id,
+    fqid_collection,
+    to_fqid,
```

Comment:
Can you name the functions analogous to the ones in https://github.com/OpenSlides/openslides-datastore-service/blob/ddd9b10c69dd98e3acf9c4fb7d024a87cad3aff0/datastore/shared/util/key_transforms.py to avoid confusion? It would be even better to use them directly, but I see no way to do that without losing our typing informations...

Reply:
(except maybe `to_fqid` since it is used so often)

Reply:
done, including to_fqid